src/libraries/System.Private.CoreLib/src/System/Decimal.DecCalc.cs

```diff
@@ -322,6 +389,14 @@ private static void Unscale(ref uint low, ref ulong high64, ref int scale)
             private static uint Div96By64(ref Buf12 bufNum, ulong den)
             {
                 Debug.Assert(den > bufNum.High64);
+
+                if (X86.X86Base.X64.IsSupported)
```

Comment:
Should code like this be surrounded with `#if TARGET_64BIT` to give smaller IL size on 32bit ?

Reply:
The trimmer handles this already.